code/mob/living/intangible/flock/flockmind.dm

```diff
@@ -138,7 +138,7 @@
 	if (!length(candidates))
 		message_admins("No ghosts responded to a Flocktrace offer from [src.real_name]")
 		logTheThing("admin", null, null, "No ghosts responded to Flocktrace offer from [src.real_name]")
-		boutput(src, "<span class='alert'>Unable to partition, please try again later.</span>")
+		flock_speak(null, "Partition failure: unable to coalesce sentience.", src.flock)
```

Comment:
Not sure if the whole flock needs to know that a partition failed, when it's announced to the flock if a partition is successful, in make_flocktrace(). If kept, should probably change the initial message to tell the flock when a partition is started.

Reply:
good point, I'll change it back to local